.thoth.yaml

```diff
@@ -18,10 +18,8 @@ managers:
   - name: version
     configuration:
       maintainers:
-        - fridex
-        - goern
         - harshad16
-        - pacospace
+        - erikerlandson
```

Comment:
As there is an OWNERS file, and kebechet takes mainainers from it if present, maybe just remove the `maintainers` section?

Although I see the proposed lists differ (maintainers in thoth.yaml vs approvers in owners). Is this intentional?

Reply:
I have updated the approver's list.